ledger/src/sigverify_shreds.rs

```diff
@@ -76,17 +73,14 @@ fn verify_shreds_cpu(
     rv
 }
 
-fn slot_key_data_for_gpu<T>(
+fn slot_key_data_for_gpu(
     thread_pool: &ThreadPool,
     batches: &[PacketBatch],
-    slot_keys: &HashMap<Slot, /*pubkey:*/ T>,
+    slot_keys: &HashMap<Slot, Pubkey>,
     recycler_cache: &RecyclerCache,
-) -> (/*pubkeys:*/ PinnedVec<u8>, TxOffset)
-where
-    T: AsRef<[u8]> + Copy + Debug + Default + Eq + std::hash::Hash + Sync,
```

Comment:
Everywhere that had `[u8; 32]` is clearly an improvement. For this one, I guess we hypothetically lose some generality. But, I guess we don't have any other objects aside from `Pubkey` that use this function, so think we can add the generic parameter back if needed.